llvm/lib/DebugInfo/Symbolize/SymbolizableObjectFile.cpp

```diff
@@ -331,7 +331,7 @@ DIGlobal SymbolizableObjectFile::symbolizeData(
   std::string FileName;
   getNameFromSymbolTable(ModuleOffset.Address, Res.Name, Res.Start, Res.Size,
                          FileName);
-  Res.DeclFile = FileName;
+  Res.DeclFile = FileName.empty() ? Res.Name : FileName;
```

Comment:
This does not look like AIX change, can you please extract into a separate patch with a test